indico/modules/events/templates/person_display.html

```diff
@@ -2,14 +2,10 @@
 
 {% macro details_table_row(caption, value) %}
     {% if value %}
-        <tr class="i-table content">
-            <td class="i-table caption">
-                {{ caption }}
-            </td>
-            <td class="i-table value">
-                {{ value }}
-            </td>
-        </tr>
+        <div>
+            <label>{{ caption }}</label>
+            <span class="text-normal">{% trans %}{{ value }}{% endtrans %}</span>
```

Comment:
And it wouldn't work properly anyway, since there is no way of extracting the actual value of `value` for translation. Text inside `trans` or `_()` must be fully static